eng/common/scripts/ChangeLog-Operations.ps1

```diff
@@ -1,4 +1,5 @@
 # Common Changelog Operations
+if (!(Test-Path Function:LogWarning)) { . "${PSScriptRoot}\logging.ps1" }
```

Comment:
Why not just always import this and SemVer.ps1 if this script needs it? I don't thing PS will care if  this gets imported more then once.

Reply:
Ok, I,ll go ahead and do that.